src/taipy/templates/scenario-management/{{cookiecutter.__root_folder_name}}/pages/scenario_page/data_node_management.py

```diff
@@ -1,94 +1,34 @@
-import re
-
-import pandas as pd
-
-
-# build partial content for all data nodes
-def build_dn_list_partial(dns, dn_contents):
-    partial_content = ""
-    for dn_id, dn_label, dn in dns.values():
-        dn_content = dn_contents.get(dn_id)
-        partial_content += f"## {dn_label}\n\n"
-        partial_content += build_dn_partial(dn, dn_label, dn_content)
-    return partial_content
-
-
 # build partial content for a specific data node
-def build_dn_partial(dn, dn_label, dn_content):
-    partial_content = ""
+def build_dn_partial(dn, dn_label):
+    partial_content = "<|part|render={selected_scenario}|\n\n"
 
     # ##################################################################################################################
     # PLACEHOLDER: data node specific content before automatic content                                                 #
     #                                                                                                                  #
     # Example:                                                                                                         #
     if dn_label == "replacement_type":
-        partial_content += "All missing values will be replaced by "
+        partial_content += "All missing values will be replaced by the data node value."
     # Comment, remove or replace the previous lines with your own use case                                             #
     # ##################################################################################################################
 
     # Automatic data node content
-    if dn_content is None:
-        partial_content += f"<center> No data for {dn_label}. </center>\n\n"
-    elif isinstance(dn_content, pd.DataFrame):
-        partial_content += "<|{contents['" + dn.id + "']}|table|rebuild|>\n\n"
-    elif (
-        isinstance(dn_content, str)
-        or isinstance(dn_content, int)
-        or isinstance(dn_content, float)
-        or isinstance(dn_content, bool)
-    ):
-        partial_content += "<center> <|{contents['" + dn.id + "']}|input|> </center>\n\n"
+    partial_content += "<|{selected_scenario.data_nodes['" + dn.config_id + "']}|data_node|>\n\n"
```

Comment:
Should add the scenario property to delete the warning: `<|{selected_scenario.data_nodes['" + dn.config_id + "']}|data_node|scenario={selected_scenario}|>`